### PR TITLE
add pomegranate margarita recipe

### PR DESCRIPTION
**Pull request by zahrakhadijha**
Add pomegranate margarita recipe to open drinks!

### CODE REVIEW

File: src/recipes/pomegranate-margarita.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
+{
+  "name": "Pomegranate Margarita",
+  "description": "A sweet and sour drink perfect for Sinco de Mayo and Summer celebrations!",
```
Comment (alfg) [inline src/recipes/pomegranate-margarita.json:3]:
```suggestion
  "description": "A sweet and sour drink perfect for Cinco de Mayo and Summer celebrations!",
```
Typo?

**Review by alfg — CHANGES_REQUESTED** [review]
One small suggestion. Otherwise, looks good.

**Comment by zahrakhadijha** [comment]
> One small suggestion. Otherwise, looks good.

Whoops! Is it fixed? I think I just fixed it in git

**Comment by alfg** [comment]
@zahrakhadijha Yep, looks good. Thank you!

**Comment by alfg** [comment]
https://opendrinks.io/recipe/pomegranate-margarita

**Pull request merged**